ci/prow/make_config.go

```diff
@@ -528,17 +620,39 @@ func generatePresubmit(title string, repoName string, presubmitConfig yaml.MapSl
 // generateGoCoveragePostsubmit generates the go coverage postsubmit job config for the given repo.
 func generateGoCoveragePostsubmit(title, repoName string, _ yaml.MapSlice) {
 	var data postsubmitJobTemplateData
+	var legacyBranches []string
 	data.Base = newbaseProwJobTemplateData(repoName)
 	data.Base.Image = coverageDockerImage
 	data.PostsubmitJobName = fmt.Sprintf("post-%s-go-coverage", data.Base.RepoNameForJob)
 	for _, repo := range repositories {
-		if repo.Name == repoName && repo.DotDev {
-			data.Base.PathAlias = "path_alias: knative.dev/" + path.Base(repoName)
+		if repo.Name == repoName {
+			if repo.DotDev {
+				data.Base.PathAlias = "path_alias: knative.dev/" + path.Base(repoName)
+			}
+			if len(repo.LegacyBranches) > 0 {
+				legacyBranches = repo.LegacyBranches
+			}
 		}
 	}
 	addExtraEnvVarsToJob(&data.Base)
 	configureServiceAccountForJob(&data.Base)
-	executeJobTemplate("postsubmit go coverage", readTemplate(goCoveragePostsubmitJob), "postsubmits", repoName, data.PostsubmitJobName, true, data)
+	if len(legacyBranches) == 0 {
+		executeJobTemplate("postsubmit go coverage", readTemplate(goCoveragePostsubmitJob), "postsubmits", repoName, data.PostsubmitJobName, true, data)
+	} else {
+		approvedBranches := data.Base.Branches
```

Comment:
I strongly suggest you keep the same naming across all the files; in this case, "branches" and "skipBranches". Using two names for the same concept makes it harder to read/understand the code.